packages/plugins/basic-pipeline/src/index.js

```diff
@@ -8,23 +8,24 @@ require('./env');
 
 class BasicPipeline {
 
-  constructor(embark, options) {
+  constructor(embark) {
     this.embark = embark;
     this.assetFiles = embark.config.assetFiles;
     this.isFirstBuild = true;
     this.embarkConfig = embark.config.embarkConfig;
     // TODO: why god why
-    this.useDashboard = options.useDashboard;
+    // this.useDashboard = options.useDashboard;
+    this.useDashboard = true;
```

Comment:
Should we get that from somewhere?

Reply:
we can't rely on the options anymore, and also having a module be aware if its' the dashboard or not was the wrong approach, so will make the assumption  that it's the dashboard which cause sit to not have the animated loading which would break in the dashboard.